projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/auto-firewall-status.tsx

```diff
@@ -2,51 +2,50 @@ import { useViewportMatch } from '@wordpress/compose';
 import { __ } from '@wordpress/i18n';
 import clsx from 'clsx';
 import useProduct from '../../../data/products/use-product';
-import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
 import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
 import { InfoTooltip } from '../../info-tooltip';
 import baseStyles from '../style.module.scss';
 import ShieldInactive from './assets/shield-inactive.svg';
 import ShieldOff from './assets/shield-off.svg';
 import ShieldSuccess from './assets/shield-success.svg';
 import { useProtectTooltipCopy } from './use-protect-tooltip-copy';
+import type { FC } from 'react';
 
-export const AutoFirewallStatus = () => {
+interface AutoFirewallStatusProps {
+	data: ProtectData;
+}
+
+export const AutoFirewallStatus: FC< AutoFirewallStatusProps > = ( { data } ) => {
```

Comment:
You'll see this pattern of passing data down through components in this PR quite a bit. I don't love this pattern, but I found it to be best in how to get the loading states of everything to be in sync and get data quicker.

In the future project to remove the window state altogether, I think we should create a `<DataProvider />` React context component so all of the components in My Jetpack can have access to the same data and loading states. Using `useSimpleQuery` everywhere works okay, but sometimes the `isLoading` states can be just a bit out of sync